dagger-common/build.gradle

```diff
@@ -62,7 +62,8 @@ dependencies {
     dependenciesCommonJar 'org.apache.flink:flink-metrics-dropwizard:' + flinkVersion
     dependenciesCommonJar 'org.apache.flink:flink-json:' + flinkVersion
     dependenciesCommonJar 'com.jayway.jsonpath:json-path:2.4.0'
-    dependenciesCommonJar 'com.gojek:stencil:2.0.15'
+    dependenciesCommonJar 'io.odpf:stencil:0.1.6'
```

Comment:
can you remove reference of old stencil from other build gradles as well? I can see a few more references